packages/flutter/lib/src/rendering/object.dart

```diff
@@ -15,6 +15,7 @@ import 'package:flutter/semantics.dart';
 import 'debug.dart';
 import 'layer.dart';
 import 'proxy_box.dart';
+import 'view.dart' show RenderView;
```

Comment:
This is still a layer violation. RenderView sits on top of object.dart.

(not sure why we have the proxy_box.dart import here, it should be removed as well)